src/graphql/schema/collection/createConnectionFieldFromPaginator.ts

```diff
@@ -0,0 +1,90 @@
+import { GraphQLFieldConfig, GraphQLInputObjectType, GraphQLInputFieldConfig } from 'graphql'
+import { Collection, Condition, conditionHelpers, NullableType, ObjectType, Paginator, Relation } from '../../../interface'
+import { formatName, buildObject, memoize2 } from '../../utils'
+import BuildToken from '../BuildToken'
+import getGqlType from '../getGqlType'
+import createConnectionGqlField from '../connection/createConnectionGqlField'
+import transformGqlInputValue from '../transformGqlInputValue'
+
+const getConditionGqlTypeAndConditionFieldEntries = memoize2((buildToken: BuildToken, type: ObjectType) => {
```

Comment:
Yeah, let’s remove `createConnectionFieldFromPaginator` and make this the single export (it’s okay if we aren’t 100% DRY, duplicated code is better than the wrong abstraction :wink:).

Also, could we rename this to `getConditionGqlType` (and consequently the file as well)?